src/main/scala/uk/gov/hmrc/SbtArtifactory.scala

```diff
@@ -66,10 +68,13 @@ object SbtArtifactory extends sbt.AutoPlugin{
 
   override val projectSettings: Seq[Def.Setting[_]] = Seq(
     publishMavenStyle := !sbtPlugin.value,
-    bintrayOrganization := Some("hmrc"),
+    bintrayOrganization := Some(maybeBintrayOrg.map { org =>
+      sLog.value.info(s"SbtArtifactoryPlugin - Resolved Bintray organisation to be '$org'")
+      org
+    }.getOrElse("hmrc")),
```

Comment:
`.orElse(Some("hmrc"))` rather than wrapping all in Some again?
Should we log when it uses the default value (e.g. move the map to the end) since we may have provided the wrong system prop key or something?